x/permission/types/keys.go

```diff
@@ -21,6 +23,8 @@ const (
 
 	// MemStoreKey defines the in-memory store key
 	MemStoreKey = "mem_permission"
+
+	FormatTimeBytesLength = 29 // len(sdk.FormatTimeBytes(time.Now()))
```

Comment:
I doubt this, `sdk.FormatTimeBytes(time.Now())` returns string, to get length of a string does not mean it is a fixed length.  Please use more determinative method.

Reply:
This sdk method is meant to get a fixed length format time string. I have double confirmed that it's fixed